### ISSUE TITLE
[COGNITION]: Pittsburgh Sidenav - The links with accordions have two different tab stops and two different links/actions

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#3445]
## Description
**[COGNITION]: Pittsburgh Sidenav - The links with accordions have two different tab stops and two different links/actions**

The paired label ~~and the carat button have two different actions,~~ and the up/down carat is not clear what the action is. [WCAG [2.4.4](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html), [2.4.9](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-link.html)]

## Type of Issue
Template

## Point of Contact

Jennifer

## Acceptance Criteria

As a user, I want to navigate the sidenav content and clearly understand what options are available before I click on an item, so that I may access content~~ and not accidentally go to a page I did not expect to and have to correct the error~~.

## Environment

* Operating System: all
* Browser: all

## Steps to Recreate

1. Enter `https://staging.va.gov/pittsburgh-health-care/` in browser
2. Clearly understand if the sidenav links with the up/down carats are one action or two

[This ticket was impacted by work that occurred while the audit was happening.]

~~1. Enter `https://staging.va.gov/pittsburgh-health-care/` in browser
2. Select `Health Services` under **Services and Locations**
3. Go back using the browser back button
4. Select `^` to the right of **Locations**
5. Select `Locations`
6. Go back using the browser back button
7. Tab through the page, noting the two tab stops in the sidenav options that have the `^`~~

## Possible Fixes (optional)

1. Rather than the carat, the +/- option may be clearer as documented in the [va design - accordions](https://design.va.gov/components/accordions).
2. For the links with accordions, make the label and symbol one tab stop and one action.
3. Visually tie the link word and the carat together to make it clear they are one action.

## WCAG or Vendor Guidance (optional)

[WCAG 2.4.4 Link purpose in context](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html)

[WCAG 2.4.9 Navigation mechanisms, links](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-link.html)]

## Screenshots or Trace Logs

![pittsburgh-sidenav-1.png](https://images.zenhubusercontent.com/5d6685f1a77bcf04870bc81c/d864b4e5-7519-425c-9865-7a0e9f6f9dbf)

**Comment by davidconlon** [comment]
@jenstrickland @rtwell 
Can you confirm that this issue is/will be resolved through implementation of the revised design?

**Comment by rtwell** [comment]
i believe so, yes these should be addressed.

**Comment by mmiddaugh** [comment]
@rtwell to add design specs
up/down arrows to be removed so that sections are either a link or a toggle, but not both. Single unti should not have two functions.

**Comment by mmiddaugh** [comment]
Hold on this one until design review complete

**Comment by mmiddaugh** [comment]
@jenstrickland is this issue now resolved by the absence of the carats in the new design?

**Comment by jenstrickland** [comment]
Yes, this will be resolved by the new design.

**Comment by mmiddaugh** [comment]
New design no longer incorporates carats. This is resolved.
